be/src/common/config.h

```diff
@@ -283,8 +281,7 @@ CONF_mInt32(cumulative_compaction_skip_window_seconds, "30");
 CONF_mInt64(min_compaction_failure_interval_sec, "600"); // 10 min
 
 // This config can be set to limit thread number in compaction thread pool.
-CONF_mInt32(min_compaction_threads, "10");
-CONF_mInt32(max_compaction_threads, "10");
+CONF_Int32(max_compaction_threads, "10");
```

Comment:
Now, `max_threads` and `min_threads`of Thread Pool can not be adjusted dynamically. Recently, I am looking into the implementation of the ThreadPool, and I consider supporting dynamic adjustment of `max_threads` and `min_threads`for thread pool later. How do you like it?

Reply:
It's OK for me~
But I think the `min_compaction_threads` is meaning less for user, we should reduce unnecessary configuration items as much as possible

Reply:
> It's OK for me~
> But I think the `min_compaction_threads` is meaning less for user, we should reduce unnecessary configuration items as much as possible

OK.